modules/rhcos-about.adoc

```diff
@@ -82,26 +96,86 @@ For {op-system} systems, the layout of the `rpm-ostree` file system has the
+add services or other user-level features to your cluster, consider adding them as
+Kubernetes workload objects. Keeping those features outside of specific node
+configurations is the best way to reduce the risk of breaking the cluster on
+subsequent upgrades.
+
+* **Day-2 customizations**: If possible, bring up a cluster without making any
+customizations to cluster nodes and make necessary node changes after the cluster is up.
+Those changes are easier to track later and less likely to break updates.
+Creating MachineConfigs or modifying Operator custom resources
+are ways of making these customizations.
+
+* **Day-1 customizations**: For customizations that you must implement when the
+cluster first comes up, there are ways of modifying your cluster so changes are
+implemented on first boot.
+Day-1 customizations can be done through Ignition configs and manifest files
+during `openshift-install` or by adding boot options during ISO installs
+provisioned by the user.
+
+Here are examples of customizations you could do on day-1:
+
+* **Kernel arguments**: If particular kernel features or tuning is needed on nodes when the cluster first boots.
+
+* **Disk encryption**: If your security needs require that the root filesystem on the nodes are encrypted, such as with FIPS compliance.
+
+* **Kernel modules**: If a particular hardware device, such as a network card or video card, does not have a usable module available by default in the Linux kernel.
+
+* **Chronyd**: If you want to provide specific clock settings to your nodes,
+such as the location of time servers.
+
+To accomplish these tasks, you can augment the `openshift-install` process to include additional
+objects such as MachineConfigs.
+Those procedures that result in creating MachineConfigs can be passed to the Machine Config Operator
+after the culster is up.
+
+
+[NOTE]
+====
+The Ignition config files that the installation program generates contain certificates that expire after 24 hours. You must complete your cluster installation and keep the cluster running for 24 hours in a non-degraded state to ensure that the first certificate rotation has finished.
+====
+
+[id="rhcos-deployed_{context}"]
+== How {op-system} systems are deployed
+
+Differences between {op-system} deployments for {product-title} are based on
+whether you are deploying on an infrastructure provisioned by the installer or by the user:
+
+* **Installer provisioned**: Some cloud environments offer pre-configured infrastructures
+that allow you to bring up an {product-title} cluster with minimal configuration.
+For these types of deployments, you can supply Ignition configs
+that place content on each node so it is there when the cluster first boots.
+
+* **User provisioned**: If you are provisioning your own infrastructure, you have more flexibility
+in how you add content to a {op-system} node. For example, you could add kernel
+options when you boot the {op-system} ISO installer to install each system.
```

Comment:
"kernel options" has not been used before in these docs; I'd suggest using "kernel arguments"

Reply:
Okay. I changed it to "kernel arguments".